apps/onboarding/src/migrations/1607954006850-session.ts

```diff
@@ -1,49 +1,48 @@
-import {MigrationInterface, QueryRunner, Table} from "typeorm"
+import { MigrationInterface, QueryRunner, Table } from 'typeorm'
 
 /* tslint:disable */
 export class session1607954006850 implements MigrationInterface {
-
```

Comment:
Are these just linting/style changes?

Reply:
Sorry, missed this. My linter does this weird thing with some files, but no changes were intended on this file, I'll revert